crates/goose/src/prompts/gooseling.md

```diff
@@ -0,0 +1,14 @@
+Based on our conversation so far, could you create:
```

Comment:
I think we'll need to refine this at some point. For most cases the instructions that it generates are really specific to the chat (if you reviewed a pr, it's specific to the pr), and I end up deleting/starting from scratch. We'll want to experiment with prompts to make it more high level maybe